src/main/java/Circle.java

```diff
@@ -0,0 +1,36 @@
+public class Circle extends Figure implements FigureInterface {
```

Comment:
```suggestion
public class Circle extends Figure {
```

if Figure class implements FigureInterface than all it's children also implements FigureInterface